public/locales/fr/common.json

```
@@ -0,0 +1,56 @@
+{
+  "scrobble": "Scrobble",
+  "getScrobbling": "Début du scrobbling",
+  "loading": "Chargement...",
+  "artist": "Artiste",
+  "song": "Musique",
```

"Song" will be used in a context where it depicts a single piece of music. I'm not a native speaker, but perhaps _chanson_ may be a better fit?

```suggestion
  "song": "Chanson",
```